.travis.yml

```diff
@@ -1,23 +1,35 @@
-matrix:
-  include:
-    # works on Precise and Trusty
-    - os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-6
-      env:
-        - MATRIX_EVAL="CC=gcc-6 && CXX=g++-6"
+language: cpp
+sudo: false
+dist: trusty
 
-language: c++
-compiler: gcc
+os:
+- linux
 
-before_install:
- - eval "${MATRIX_EVAL}"
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-6
+    - lcov
+
+after_success:
+# Source: https://github.com/codecov/example-cpp11-cmake
+- cd ${TRAVIS_BUILD_DIR}
+- lcov --directory . --capture --output-file coverage.info
+- lcov --remove coverage.info '/usr/*' --output-file coverage.info
+- lcov --remove coverage.info 'extern/catch2/*' --output-file coverage.info
+- lcov --list coverage.info
+- bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
 
 git:
   submodules: false
 
-script: cmake components/scream && make && make test
+branches:
```

Comment:
@mfdeakin-sandia , can you explain this? We don't want to limit testing to master, right? We want to test PRs as they come in.